client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/SchoolEventForm.tsx

```diff
@@ -12,13 +12,19 @@ import InteractionEventDialogData from 'models/Contexts/InteractionEventDialogDa
 import {InteractionEventDialogContext} from '../../InteractionsEventDialogContext/InteractionsEventDialogContext';
 import InteractionEventDialogFields from '../../InteractionsEventDialogContext/InteractionEventDialogFields';
 
-export const grades = [
+const elementarySchoolPlaceType = 15;
+const highSchoolPlaceType = 127;
```

Comment:
remove

Reply:
I must keep them, because they are used in order to choose which form I want to render